### ISSUE TITLE
[Question] Let’s use Supabase as DB

### ISSUE THREAD

**Issue by 0w9** [withmarko-community/withmarko-community#3]
Rules:
- [X] I have searched existing github issues for my answer before creating this
 
Description:
I highly suggest to use Supabase as database. They make authentication (which, afaik, our project needs) really easy as it allows to have a private table with all user data. The other, public, tables can then link to a user using „foreign keys“.

I’ve used their the service a lot lately and the auth works perfect, and is easy to setup with Google, Twitter, etc.!

**Comment by sixfourtwelve** [comment]
I agree. Supabase would be super handy as well. We can always provide dummy data as well to make development easier
